client/devdocs/design/playground.jsx

```diff
@@ -121,6 +121,7 @@ import TokenFields from 'components/token-field';
 import Tooltip from 'components/tooltip';
 import Version from 'components/version';
 import VerticalMenu from 'components/vertical-menu';
+import VerticalNav from 'components/vertical-nav';
```

Comment:
Should we also include `VerticalNavItem` here, so that the above example can work in the playground?

Reply:
Thanks @vindl, good point!